### DISCUSSION TITLE
Client crashes when trying to do lightdm autologin

### DISCUSSION THREAD

**Discussion by SimStone** [ltsp/ltsp#575] [Q&A] [answered]
Hi,

Thanks to your help I was able not to use sshfs and get my clients /home on the clients RAM (#570) but now I'm experiencing trouble while trying to get my clients to autologin using an account pierre configured using "LIGHTDM_CONF" this way :

LIGHTDM_CONF="user-session=xfce
autologin-user=pierre
autologin-user-timeout=5"

In this configuration on the lightdm screen I see the user pierre in the user field and it stays on this screen for 5 seconds so the parameters are taken into account but after 5 seconds the client crashes and gets back to lightdm screen for 5 seconds and loops that way.

For testing I tried to disable autologin by commenting the LIGHTDM_CONF entry and was successfully able to log in using the account. 
Since I'm able to log in with this account without autologin there must be something I'm doing wrong in the lightdm autologin settings configuration.

**Comment by alkisg** [comment, marked as answer]
Maybe autologin bypasses PAM so you need to manually create the /home/username directory, and give it appropriate permissions.
E.g.: `POST_INIT_MK_HOME="mkdir /home/pierre && chown pierre:pierre /home/pierre"`
